src/ostorlab/runtimes/local/models/alembic/versions/69c7d2f49a4e_switch_asset_types_to_enum.py

```diff
@@ -0,0 +1,61 @@
+"""Switch asset types to enum
+
+Revision ID: 69c7d2f49a4e
+Revises: b31a27c8584f
+Create Date: 2024-06-26 18:21:23.093382
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "69c7d2f49a4e"
+down_revision = "b31a27c8584f"
```

Comment:
do there have to be lower case?

Reply:
!!! this is the migration file, and its autogenerated !